delphi-frontend/src/test/java/au/com/integradev/delphi/type/intrinsic/IntrinsicArgumentMatcherTest.java

```diff
@@ -83,6 +84,14 @@ void testAnyOrdinal() {
     assertThat(matches(ANY_ORDINAL, FACTORY.getIntrinsic(IntrinsicType.STRING))).isFalse();
   }
 
+  @Test
+  void testAnyTypedPointer() {
+    Type typedPointer = FACTORY.pointerTo("PInteger", FACTORY.getIntrinsic(IntrinsicType.INTEGER));
+    assertThat(matches(ANY_TYPED_POINTER, typedPointer)).isTrue();
+    assertThat(matches(ANY_TYPED_POINTER, FACTORY.nilPointer())).isFalse();
+    assertThat(matches(ANY_TYPED_POINTER, FACTORY.nilPointer())).isFalse();
```

Comment:
Duplicate line - did you mean to test `FACTORY.untypedPointer()` in the second case here?